src/com/amazon/ion/benchmark/WriteRandomIonValues.java

```diff
@@ -50,6 +51,14 @@
     final static private IonSystem system = IonSystemBuilder.standard().build();
     final static private List<Integer> defaultRange = WriteRandomIonValues.parseRange("[0, 1114111]");
     final static private Timestamp.Precision[] precisions = Timestamp.Precision.values();
+    static private int codePointLength = 0;
+    static private List<Integer> pointRange;
+    static private List<Integer> expValRange;
+    static private List<Integer> coefficientDigitRange;
+    static private String timestampTemplate;
+    static private IonStruct fields;
+    static private Map<String, Object> annotationMap;
+    static private IonStruct constraintStruct;
```

Comment:
I like the idea here to improve readability. Since you never assign these, I'd make two small changes:
1. Make them `final` and explicitly assign them to `null`.
2. Name them like `NO_CODE_POINT_LENGTH` or `NO_FIELDS` (using the naming style for constants).

This makes it clear that they're intended to be used to improve readability and don't need to be assigned.